packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -223,12 +248,18 @@ contract ChugSplashManager is Owned {
     }
 
     /**
-     * @notice Allows the owner to approve a bundle to be executed. Note that the bundle can be
-     *         executed as soon as the bundle is approved.
+     * @notice Allows the owner to approve a bundle to be executed. Owners must submit a bond of
+     *         `ownerBondAmount`, which will be forfeited if they cancel an active bundle. Note that
+     *          the bundle can be executed as soon as the bundle is approved.
      *
      * @param _bundleId ID of the bundle to approve
      */
-    function approveChugSplashBundle(bytes32 _bundleId) public onlyOwner {
+    function approveChugSplashBundle(bytes32 _bundleId) public payable onlyOwner {
+        require(
```

Comment:
We may actually be able to make this slightly cleaner (and separate concerns better) by doing the following:
- Add a function `depositETH` that allows anyone to send funds to the contract
- Add a function `withdrawETH` that allows the owner to withdraw funds to any recipient address
- Funds cannot be withdrawn during an active upgrade
- Funds are used to pay executors
- We can simply check that `balance(this) >= BOND_AMOUNT` when calling `approveChugSplashBundle`, no need for the owner to also hold ETH (some separate account can be responsible for that)

Reply:
Kills two birds with one stone. We'll also later have to put some logic in the executor to simulate the transactions, figure out how much gas the whole upgrade would require, and check if the account has sufficient ETH to be worth claiming the upgrade.

Reply:
> We may actually be able to make this slightly cleaner (and separate concerns better) by doing the following:
> 
> * Add a function `depositETH` that allows anyone to send funds to the contract
> * Add a function `withdrawETH` that allows the owner to withdraw funds to any recipient address
> * Funds cannot be withdrawn during an active upgrade
> * Funds are used to pay executors
> * We can simply check that `balance(this) >= BOND_AMOUNT` when calling `approveChugSplashBundle`, no need for the owner to also hold ETH (some separate account can be responsible for that)

Sounds good. Added this except for the second to last bullet point, which seems out of scope for this PR since it involves executor logic in the ChugSplashManager.